graders/python/python_autograder/pl_result.py

```diff
@@ -13,20 +13,9 @@ class PLTestResult(unittest.TestResult):
     """
 
     error_message = (
-        "The grading code failed -- sorry about that!\n\n"
-        "This may be an issue with your code.\nIf so, you can "
-        "take a look at the traceback below to help debug.\n"
-        "If you believe this is an issue with the grading code,\n"
-        "please notify the course staff.\n\n"
-        "The error traceback is below:\n"
-    )
-
-    major_error_message = (
-        "The grading code was not able to run.\n"
-        "Please notify the course staff "
-        "and include this entire message,\n"
-        "including the traceback below.\n\n"
-        "The error traceback is:\n"
+        "There was an error while grading your code.\n\n"
+        "Review the question text to ensure your code matches the expected requirements, such as variable names, function names, and parameters.\n\n"
```

Comment:
I believe this line may be too long and may cause the output box to scroll unnecessarily (it doesn't wrap).

Reply:
Good call. Wrapped in af83690ca472d0d7d9fd78d59b9b6cd26806e41a